contents/blog/best-open-source-feature-flag-tools.md

```diff
@@ -32,54 +31,51 @@ In this guide, we’ll explore some of the best open source feature flag tools,
 
 If you want to run experiments as well, we recommend reading our guide to the [best open-source A/B testing tools](/blog/best-open-source-ab-testing-tools).
 
-## 1. [PostHog](https://posthog.com)
+## 1. PostHog
 
 ![PostHog](../images/blog/open-source-feature-flags/posthog-feature-flags.png)
 
-PostHog is an all-in-one [Product OS](https://posthog.com/product) that captures and visualizes all of the data your team needs in one place. Complete with product analytics, feature flags, session recording, heatmaps, funnels and more, it’s a great choice if you want to build a comprehensive, data-driven product strategy, fast.
+[PostHog](https://posthog.com) is an all-in-one [Product OS](https://posthog.com/product) that captures and visualizes all the data your team needs in one place. Complete with product analytics, feature flags, session recording, heatmaps, funnels and more, it’s a great choice if you want to build a comprehensive, data-driven product strategy, fast.
 
 PostHog's [feature flags](https://posthog.com/docs/user-guides/feature-flags) also power its [experimentation suite](https://posthog.com/docs/user-guides/experimentation), so you can test product changes with A/B tests or define user segments for feature access. [Multivariate flags](https://posthog.com/docs/user-guides/feature-flags#multivariate-feature-flags-beta) allow you to test several variants of the same interchangeable elements.
 
+Recent additions to PostHog's feature flag product include support for JSON feature flags and the addition of feature flag support for Flutter.
+
 ### Who is PostHog for?
 
 PostHog is built primarily for engineers and other product-minded teams. With all of your analytics in one place, everyone in the organization is empowered to examine and act on user data. You can also self-host PostHog Open Source, which is ideal for any company with stringent data privacy needs.
 
-### Strengths
+### Features and benefits
 
-- All-in-one product analytics platform
+- Unlimited feature flags
+- All-in-one analytics platform
 - Easy to get set up (no SQL needed)
 - A/B tests with PostHog’s experimentation suite
 - Multivariate flags for several variations per test
 - Target by user properties and percentage
-- Unlimited flags
-- Multi-environment support
 - Free third-party apps to enrich your data
 
 ### How much does it cost?
 
-Feature flags are a core feature of all editions, including the open source release. While multivariate feature flags are limited to paid plans, PostHog Cloud is [free up to 1 million events every month](/pricing).
+Feature flags are a core feature of all editions, including the open source release. While multivariate feature flags are limited to paid plans, PostHog Cloud is [free up to 1 million events every month](/pricing) and feature flags are included at no additional cost.
```

Comment:
```suggestion
Feature flags are a core feature of all editions, including the open source release. While multivariate feature flags are limited to paid plans, PostHog Cloud is [free up to 1 million events every month](/pricing) and unlimited feature flags are included at no additional cost.
```

Reply:
Is "unlimited feature flags" appealing? I'm not fully convinced but might be worth in a comparison.